tools/bin/ci_integration_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+# runs integration and/or standard tests for an integration name
+
+connector="$1"
+all_integration_tests=$(./gradlew integrationTest --dry-run | grep 'integrationTest SKIPPED' | cut -d: -f 4)
+all_standard_python_tests=$(./gradlew standardSourceTestPython --dry-run | grep 'standardSourceTestPython SKIPPED' | cut -d: -f 4)
+
+if [[ "$connector" == "all" ]] ; then
+  echo "Running: ./gradlew --no-daemon --scan integrationTest standardSourceTestPython"
+  ./gradlew --no-daemon --scan integrationTest standardSourceTestPython
+else
+  selected_integration_test=$(echo "$all_integration_tests" | grep "^$connector$" || echo "")
```

Comment:
not blocking for this PR, but should this selection logic live in a gradle task where we just say "integrationTest" and it figures out whether it's standard python or anything else? That way it can be re-used by any other tools, and is more accessible for anyone new to the repo since it's all wrapped in a gradle task

Reply:
Agreed. It feels like `integrationTest` is the right umbrella task, but we'd need to switch that to `javaIntegrationTest` or something first to put both under it.